server/src/immich/controllers/job.controller.ts

```diff
@@ -17,7 +17,7 @@ export class JobController {
   }
 
   @Put(':id')
-  sendJobCommand(@Param() { id }: JobIdDto, @Body() dto: JobCommandDto): Promise<JobStatusDto> {
+  sendJobCommand(@Param() { id }: JobIdParamDto, @Body() dto: JobCommandDto): Promise<JobStatusDto> {
```

Comment:
Do we have to regenerate openapi for this?

Reply:
I didn't get any changes. I don't think there are "dto"s for params in any of the generated code.